src/translations/screens/subscreens/TicketTraveller.ts

```diff
@@ -95,12 +96,40 @@ function generic(travellerType: string) {
   }
 }
 
+function specificUserProfileDescription(
+  travellerType: string,
+  ticketType: string | undefined,
+) {
+  if (ticketType === undefined) return false;
+  switch (ticketType) {
+    case 'travel-pass':
+      if (travellerType === TravellerType.adult) {
+        return _('Over 18 år', 'Age 18 or older');
+      } else if (travellerType === TravellerType.child) {
+        return _('Til og med 17 år', 'Age 17 or younger');
+      }
+      return false;
+    default:
+      return false;
+  }
+}
+
 const TicketTravellerTexts = {
   information: (travellerType: string, ticketType: string | undefined) => {
     return (
       specificOverrides(travellerType, ticketType) || generic(travellerType)
     );
   },
+  userProfileDescription: (
```

Comment:
Maybe we should rename this to `productTypeSpecificUserProfileDescription` or something similar to be clear that this is an override of user profile descriptions for some products? 🤔

Reply:
Better like this? 

https://github.com/AtB-AS/mittatb-app/pull/3628/commits/72d8c8dcea876e6e821dd74e0d265cf8f23ebb68

Reply:
Yes! Nice 👍